Content.Server/Sound/EmitSoundSystem.cs

```diff
@@ -55,6 +57,15 @@ private void HandleEmitSoundOnTrigger(EntityUid uid, EmitSoundOnTriggerComponent
         args.Handled = true;
     }
 
+    private void HandleEmitSoundOnInteractUsing(EntityUid uid, EmitSoundOnInteractUsingComponent component, InteractUsingEvent args)
+    {
+        var curUsedItemID = Prototype(args.Used)?.ID;
+        if (component.UsedItemID == curUsedItemID)
+        {
+            TryEmitSound(uid, component, args.User, false);
+        }
+    }
```

Comment:
Do not hardcode prototype ID checks, they are only used for spawning and nothing else.

Reply:
the implementation is kinda funky anyway, probably better to use a tag or component check instead

Reply:
I'm not sure what the exact issue is here? Is using prototype IDs prohibited or is the manner I retrieved the ID prohibited?

Reply:
I'm not sure what the exact issue is here? Is using prototype IDs prohibited or is the manner I retrieved the ID prohibited?

Reply:
Oh wait, just saw the "Don't Cache Prototypes" rule on the dev wiki
Apologies, I will figure out how to do this without caching then.

Reply:
I've changed it to compare prototypes instead of strings. Is this what you requested or am I confused with the issue?